tmol/database/default/chemical/chemical.yaml

```diff
@@ -1988,3 +2232,61 @@ residues:
         pH: 7
       virtual: []
     chi_samples: []
+
+variants:
+  - name:  CarboxyTerminus
+    display_name: cterm
+    pattern: '[*][*]C(=O)[{up}]'
+    remove_atoms:
+    - <{up}>
+    - <O1>
+    add_atoms:
+    - { name: O    ,  atom_type: OOC }
+    - { name: OXT  ,  atom_type: OOC }
+    modify_atoms: []
+    add_connections: []
+    add_bonds:
+    - [   <C1>,    OXT ]
+    - [   <C1>,    O ]
+    icoors:
+    - { name:     O, source: <O1>, phi:   80.0 deg, theta: 60.0 deg, d: 1.2, parent:  <C1>, grand_parent: <*2>, great_grand_parent: <*1>}
+    - { name:   OXT, source: <O1>, phi: -180.0 deg, theta: 60.0 deg, d: 1.2, parent:  <C1>, grand_parent: <*2>, great_grand_parent: O }
+
+  - name:  AminoTerminus
+    display_name: nterm
+    pattern: '[*][*][NH][{down}]'
```

Comment:
No there isn't differentiation of the two.  For now it will return atom w/ lower id first though (making it consistent between AAs).  We can change this to be more specific on atom type (element+protonation state, not "atom type"), but not precisely on C vs. CB).    There are a few options we can do:
1) Something like O=CC[NH][{down}] would be more specific to alpha AAs (though some NCAA might hit sidechain atoms)
2) Add a tag preferring/forcing an atom name match ([*:C][*:CA][NH][{down}] maybe?